roles/cnf_cert/defaults/main.yml

```diff
@@ -1,9 +1,9 @@
 ---
 tnf_stage: "tests"
 tnf_check_commit_sha: false
-test_network_function_repo: "https://github.com/test-network-function/cnf-certification-test"
-test_network_function_version: v5.2.2
-test_network_function_project_name: cnf-certification-test
+redhat_best_practices_for_k8s_repo: "https://github.com/test-network-function/certsuite"
```

Comment:
I see there's a redirect for the old repo, I think is safe to leave it unchanged:

```Shell
❯ curl -sI https://github.com/test-network-function/cnf-certification-test | grep ^location
location: https://github.com/redhat-best-practices-for-k8s/certsuite
```